### ISSUE TITLE
Warning - Animated : UseNativeDriver was not specified

### ISSUE THREAD

**Issue by juulieen** [BugiDev/react-native-calendar-strip#173]
### Environement 

react-native: 0.62.2
react-native-calendar-strip: 1.4.1

### Description

I updated to react 0.62.2 and have a new Warning : 

```
Animated: `useNativeDriver` was not specified. This is a required option and must be explicitly set to `true` or `false`
```

> The useNativeDriver property is now required when using the Animated API. The previous default was false so setting it to that preserves the previous behaviour, however, if you're animating a transform then you'll likely get better performance by setting it to true. - [Upgrade to React Native 0.62](https://reactnative.thenativebits.com/courses/upgrading-react-native/upgrade-to-react-native-0.62/)

### Proposal

Maybe to fix this we can simply add a `prop` to the components with a default `value` to false to keep the default behaviour

**Comment by peacechen** [comment]
Thanks for noticing this.  `useNativeDriver` is stable enough to make that the default setting.  Would you mind creating a PR for this?
FYI props default to true:
https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true

**Comment by peacechen** [comment]
This fix has been released in 1.4.2.
